### PR TITLE
fix(aws_builder): change the builders to configure gp3 root disks

### PR DESCRIPTION
**Pull request by fruch**
update the launch templates to use gp3 disk,

and introduce code that can create new launch template version and update them to be the default, before this change the only check we did if the launch template existed or not, and never update anything with those templates
#### Testing
- [x] apply it on one region and test
- [x] apply it to all regions

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@soyacz @vponomaryov @yaronkaikov   for awareness that i'm gonna update all of the ASGs SCT is using, so it might have some effect I didn't know about.

I did the change for eu-west-1 (and for the ASG used by SCT CI)
After feedback on the changes here, I'll make it on all regions (and merged it)

**Review by yaronkaikov — APPROVED** [review]

File: sdcm/utils/aws_builder.py [hunk 1]
```diff
@@ -114,104 +114,59 @@ def name(self):
...
-        instances = self.region.client.describe_instances(Filters=[{"Name": "tag:Name",
-                                                                    "Values": [self.name]},
-                                                                   {"Name": "instance-state-name",
-                                                                    "Values": ["running"]}])
-
-        if instances["Reservations"] and instances["Reservations"][0]["Instances"]:
-            return self.region.resource.Instance(instances["Reservations"][0]["Instances"][0]['InstanceId'])
-
-        tags = {
-            "bastion": "true",
```
Comment (soyacz) [inline sdcm/utils/aws_builder.py:135]: so we no longer use builders as bastions?

**Review by vponomaryov — APPROVED** [review]
LGTM

File: sdcm/utils/aws_builder.py (hunk 1 above)
Comment (fruch) [reply sdcm/utils/aws_builder.py:135]: we use the sct-runner available on that region, if we'll see it's not enough, we'll thing if we want few machines like that, but they won't be connect to jenkins.

Comment (fruch) [reply sdcm/utils/aws_builder.py:135]: in GCP so far we don't have any, no one complain about it. (so far)

**Pull request merged**